tests/test_all_fields.py

```diff
@@ -38,13 +38,6 @@
         'price',
         'updated',
     },
-    'invoices':{
-        'payment_settings',
-        'on_behalf_of',
-        'custom_fields',
-        'automatic_tax',
-        'quote',
-    },
```

Comment:
Can't we use time convertor and remove `finalized_at` and `created` from KNOWN_FAILING_FIELDS

Reply:
changed the format for created, `finalized_at` is deprecated, hence it is skipped.